component/admin/language/en-GB.com_jevents.sys.ini

```diff
@@ -83,3 +83,11 @@ JEVENTS="JEvents"
 
 COM_JEVENTS_LIST_DEFAULT_TITLE="List of Events"
 COM_JEVENTS_LIST_DEFAULT_DESC="List of Events where you can choose the columns to display"
+
+; Float Theme language strings ##
+PLG_INST_JEVENTS_JEVFLOAT="JEvents Float Layout Installed"
+PLG_INST_JEVENTS_JEVFLOAT_SUCC2="You have successfully installed the JEvents Float Layout"
+PLG_INST_JEVENTS_JEVFLOAT_SUCC1="You have successfully update the JEvents Float Layout"
+JEV_EVENT_STARTED="Event Started"
```

Comment:
Because sys.ini is called on Install whereas .ini is not.
On Wed, 4 May 2016 at 16:58, Geraint Edwards <notifications@github.com>
wrote:

> In component/admin/language/en-GB.com_jevents.sys.ini
> <https://github.com/JEvents/JEvents/pull/323#discussion_r62065838>:
>
> > @@ -83,3 +83,11 @@ JEVENTS="JEvents"
> >
> >  COM_JEVENTS_LIST_DEFAULT_TITLE="List of Events"
> >  COM_JEVENTS_LIST_DEFAULT_DESC="List of Events where you can choose the columns to display"
> > +
> > +; Float Theme language strings ##
> > +PLG_INST_JEVENTS_JEVFLOAT="JEvents Float Layout Installed"
> > +PLG_INST_JEVENTS_JEVFLOAT_SUCC2="You have successfully installed the JEvents Float Layout"
> > +PLG_INST_JEVENTS_JEVFLOAT_SUCC1="You have successfully update the JEvents Float Layout"
> > +JEV_EVENT_STARTED="Event Started"
>
> Why did you move this from .ini to .sys.ini ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JEvents/JEvents/pull/323/files/d5d69a51a293630b4205b039e3b4ce193edcdaf3#r62065838>
>